cmd/apiserver-boot/boot/create/resource.go

```diff
@@ -54,6 +55,7 @@ apiserver-boot create group version resource --group insect --version v1beta1 --
 func AddCreateResource(cmd *cobra.Command) {
 	RegisterResourceFlags(createResourceCmd)
 
+	createResourceCmd.Flags().StringVar(&shortName, "short-name", "", "if set, add a short name for the resource, exposed in API discovery documents, and used by clients to support invocations like 'kubectl get <shortname>'. It must be all lowercase.")
```

Comment:
"if set, add a short name for the resource, ~exposed in API discovery documents, and used by clients to support invocations like 'kubectl get <shortname>'.~ It must be all lowercase."